scanpy/_settings.py

```diff
@@ -303,9 +303,9 @@ def cache_compression(self, cache_compression: Optional[str]):
     @property
     def max_memory(self) -> Union[int, float]:
         """\
-        Maximal memory usage in Gigabyte.
+        Maximum memory usage in Gigabyte.
 
-        Is currently not well respected....
+        Is currently not well respected…
```

Comment:
Should we just remove it all together? It seems to be more confusing than helpful.